imagespace_weapons/server/__init__.py

```diff
@@ -24,7 +25,18 @@ def add_maintype_to_qparams(event):
     event.addResponse(event.info)
 
 
+def uppercase_basename_for_resourcenames(event):
```

Comment:
Would be good to document in a comment why this is needed (as hacky as it may be).